graph/config/cytoscape/cytoscape.go

```diff
@@ -281,6 +283,18 @@ func buildConfig(trafficMap graph.TrafficMap, nodes *[]*NodeWrapper, edges *[]*E
 			}
 		}
 
+		// node may represent an Istio Egress Gateway
+		if gateways, ok := n.Metadata[graph.IsEgressGateway]; ok {
```

Comment:
nit: Maybe this could be `else if` to make it more clear that a node is either an ingress or an egress but not both?

Reply:
Yes, let me do it.

Reply:
Done!